README.md

```diff
@@ -1,7 +1,6 @@
 ![now](https://assets.zeit.co/image/upload/v1581518533/repositories/now-cli/v4.png)
 
 [![CI Status](https://badgen.net/github/checks/zeit/now?label=CI)](https://github.com/zeit/now/actions?workflow=CI)
-[![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/zeit)
```

Comment:
There are many places in this repo that reference `spectrum.chat`.

These will all need to be changed. Perhaps we could link to GitHub Discussions instead?

Let me find a good looking badge.

Reply:
https://badgen.net/badge/join%20the%20discussion/on%20github/black?icon=github

![img](https://badgen.net/badge/join%20the%20discussion/on%20github/black?icon=github)